server/Common/Models/DTOs/LoginRequestDto.cs

```diff
@@ -1,8 +1,11 @@
-﻿namespace Server.API.DTOs;
+﻿using System.ComponentModel.DataAnnotations;
 
-public class LoginDto
+namespace Common.Models.DTOs;
+
+public class LoginRequestDto
 {
     public string? Username { get; set; }
```

Comment:
Talked about Username needed to be UserName for functionality, not sure if this needs to be UserName.

Reply:
not in the dto.  Sometimes the Entities get weird with particular naming conventions but DTOs will be fine.